BootloaderCommonPkg/Include/Library/FirmwareUpdateLib.h

```diff
@@ -41,6 +41,10 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define CAPSULE_FLAGS_CFG_DATA  BIT0
 
+#define FW_UPDATE_COMP_SBL  SIGNATURE_32('B', 'I', 'O', 'S')
+#define FW_UPDATE_COMP_CSME SIGNATURE_32('C', 'S', 'M', 'E')
+#define FW_UPDATE_COMP_CSMD SIGNATURE_32('C', 'S', 'M', 'D')
+

```

Comment:
How about update FW_UPDATE_COMP_SBL to FW_UPDATE_COMP_BIOS_REGION to match with its signature "BIOS"?
and update FW_UPDATE_COMP_CSME to be FW_UPDATE_COMP_CSME_REGION
FW_UPDATE_COMP_CSMD to FW_UPDATE_COMP_CSME_DRIVER?

Reply:
Okay I will change

Reply:
You acted quickly while I testing and reviewing. 
To make it shorter, better to use FWU_COMP_BIOS_REGION if you want :)

Reply:
Done

Reply:
There are many type casts to check SIGNATURE_32 value. What about using SIGNATURE_64 here?

Reply:
We will use the same 32 byte signature for all components inside the flash map, even though we might change these to 64 but type cast is still required to match the signature to components from flash map.

We also cannot reduce the hardware instance to 32 bytes because we have container name : component name format. Higher bytes will have name of component inside container and lower bytes will have component name